cmd/clusterctl/examples/aws/generate-yaml.sh

```diff
@@ -2,30 +2,27 @@
 set -o errexit
 set -o nounset
 
+# Directories.
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
-
 OUTPUT_DIR=${DIR}/out
 
-# provider-components
-export CLUSTER_CONTROLLER_IMAGE="${CLUSTER_CONTROLLER_IMAGE:-gcr.io/k8s-cluster-api/aws-cluster-controller:0.0.1}"
-export MACHINE_CONTROLLER_IMAGE="${MACHINE_CONTROLLER_IMAGE:-gcr.io/k8s-cluster-api/aws-machine-controller:0.0.1}"
+# Manager image.
+export MANAGER_IMAGE="${MANAGER_IMAGE:-gcr.io/k8s-cluster-api/aws-cluster-controller:0.0.1}"
```

Comment:
`MANAGER_IMAGE` is supplied here and also in the makefile.
Can we supply this value in one place?

Reply:
maybe error out if `MANAGER_IMAGE` is not set when this script runs.

Reply:
`MANAGER_IMAGE` in the `Makefile` actually sets this variable when the Makefile runs.